src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  const [isDisabled, setIsDisabled] = useState(true);
+  const buttonText = ["Job Opportunities", "Trainings / Workshops"];
+
+  const selectOpportunityButton = () => {
+    if (isDisabled === true) {
```

Comment:
this needs some comments. i dont get why the select opportunity button gets disabled and why, when its disabled, its related to "trainings", and when its not disabled, its related to "job opportunities". put the comment above the react state

Reply:
This function is used to check if the button is disabled or not.

The default state is the jobs button is disabled and jobs data is fetched. So when the training button is clicked it detects the disabled button (jobs) and fetches the opposite data which is training.

As a result, the training button becomes disabled and un-clickable and training data is fetched. So on the next click of the jobs button, it checks that the training button is disabled and fetches the opposite data. and so on.

Reply:
@louisrli I added comments, and fixed the mentioned issues.